etc/cmake/options.cmake

```diff
@@ -317,6 +317,16 @@ if(OT_UDP_FORWARD)
     target_compile_definitions(ot-config INTERFACE "OPENTHREAD_CONFIG_UDP_FORWARD_ENABLE=1")
 endif()
 
+option(OT_PACKAGE_NAME "enable vendor package name")
```

Comment:
Should we use `variable` but not `option`? `option` is designed for boolean switches https://cmake.org/cmake/help/v3.10/command/option.html#option.

And better to have `set(OT_PACKAGE_NAME ...)` at beginning of `CMakeListstxt` but not this `options.cmake` file.

Reply:
Should be fixed now, thanks.